frontend/pages/blank.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
+// This page is used as a redirect URI for our popup login flow, see
+// https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md#redirecturi-considerations
+
+definePageMeta({
+  layout: 'empty'
```

Comment:
another (maybe simpler) approach? Through an index.html in some folder in the public directory. That way nuxt could never mess this up?

Reply:
I did consider it, and that would also be a  totally reasonable approach. I'm going to stick to this for a few reasons:

* The `redirectUri` actually is important in the redirect flow (we currently use the popup flow). We've seen on other projects that there are scenarios (PWAs, some mobile browsers) where popup mode doesn't play nice and you want to fallback to redirect mode, so we're going to want to be able to add little niceties to this page over time, like taking query parameters and doing stuff with them.
* I think there's utility in having the same `<head>` matter that the rest of the site has

I agree there's a minor risk that something like `middleware` causes problems with this page, but since the page is only ever shown for a fraction of a second, I think the risk is minimal.